src/main/resources/application.properties

```diff
@@ -11,4 +11,6 @@ spring.jpa.hibernate.ddl-auto=create
 
 spring.jpa.defer-datasource-initialization=true
 
-logging.level.org.hibernate.orm.jdbc.bind=TRACE
\ No newline at end of file
+logging.level.org.hibernate.orm.jdbc.bind=TRACE
+
+stripe.secret-key=sk_test_51RshaoQp3kUgTrozcwAE53fg97xuWsVnLKF0Xj8BbcYh3v7RjJaLh2QSih0zCgPhEh1A7hFew4fgz0ryDh4wQJQ2004fomMaco
```

Comment:
The key is exposed, making it very insecure. Don’t commit it and find a way to hide it. Since the previously issued key has already been leaked, please reissue it. 🔐